src/systems/hubs-systems.ts

```diff
@@ -92,83 +111,125 @@ AFRAME.registerSystem("hubs-systems", {
-    const world = APP.world;
-
-    networkReceiveSystem(world);
-    onOwnershipLost(world);
-    mediaLoadingSystem(world);
-
-    physicsCompatSystem(world);
-
-    networkedTransformSystem(world);
-
-    const systems = AFRAME.scenes[0].systems;
-    systems.userinput.tick2();
-
-    interactionSystem(world, this.cursorTargettingSystem, t, systems);
-
-    buttonSystems(world);
-    constraintsSystem(world, systems.userinput);
-
-    // We run this earlier in the frame so things have a chance to override properties run by animations
-    this.animationMixerSystem.tick(dt);
-
-    this.characterController.tick(t, dt);
-    this.cursorTogglingSystem.tick(systems.interaction, systems.userinput, this.el);
-    this.interactionSfxSystem.tick(systems.interaction, systems.userinput, this.soundEffectsSystem);
-    this.superSpawnerSystem.tick();
-    this.emojiSystem.tick(t, systems.userinput);
-    this.cursorPoseTrackingSystem.tick();
-    this.hoverMenuSystem.tick();
-    this.positionAtBorderSystem.tick();
-    // this.constraintsSystem.tick();
-    this.twoPointStretchingSystem.tick();
-
-    floatyObjectSystem(world);
-
-    this.holdableButtonSystem.tick();
-    this.hoverButtonSystem.tick();
-    this.drawingMenuSystem.tick();
-    this.hapticFeedbackSystem.tick(
-      this.twoPointStretchingSystem,
-      false,
-      false
-      // TODO: didInteractLeftThisFrame doesn't exist?
-      // this.singleActionButtonSystem.didInteractLeftThisFrame,
-      // this.singleActionButtonSystem.didInteractRightThisFrame
-    );
-    this.soundEffectsSystem.tick();
-    this.scenePreviewCameraSystem.tick();
-    this.physicsSystem.tick(dt);
-    this.inspectYourselfSystem.tick(this.el, systems.userinput, this.cameraSystem);
-    this.cameraSystem.tick(this.el, dt);
-    cameraToolSystem(world);
-    this.waypointSystem.tick(t, dt);
-    this.menuAnimationSystem.tick(t);
-    this.spriteSystem.tick(t, dt);
-    this.uvScrollSystem.tick(dt);
-    this.shadowSystem.tick();
-    videoMenuSystem(world, systems.userinput);
-    videoSystem(world, this.audioSystem);
-    mediaFramesSystem(world);
-    this.audioZonesSystem.tick(this.el);
-    this.gainSystem.tick();
-    this.nameTagSystem.tick();
-
-    deleteEntitySystem(world, systems.userinput);
-    destroyAtExtremeDistanceSystem(world);
-    removeNetworkedObjectButtonSystem(world);
-    removeObject3DSystem(world);
-
-    // We run this late in the frame so that its the last thing to have an opinion about the scale of an object
-    this.boneVisibilitySystem.tick();
-
-    networkSendSystem(world);
-  },
-
   remove() {
     this.cursorTargettingSystem.remove();
   }
 });
+
+export function mainTick(xrFrame: XRFrame, renderer: WebGLRenderer, scene: Scene, camera: Camera) {
+  const world = APP.world;
+  const sceneEl = AFRAME.scenes[0];
+  const aframeSystems = sceneEl.systems;
+  const hubsSystems = aframeSystems["hubs-systems"];
+
+  // TODO does anything actually ever pause the scene?
```

Comment:
I see that this check existed before, and has just been moved to here, but it seems like it would be a bug for anything to be able to pause the scene. At least -- I'm not aware of anything that would be doing that on purpose.

Along similar lines, perhaps we shouldn't start the `mainTick`until `DOMContentDidLoad` so we can remove the check here.